ci_framework/roles/edpm_kustomize/tasks/main.yml

```diff
@@ -0,0 +1,75 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Check mandatory parameters
+  block:
+    - name: Ensure we pass the mandatory parameters
+      ansible.builtin.assert:
+        that:
+          - cifmw_edpm_kustomize_cr_path is defined
+        msg: "Please ensure cifmw_edpm_kustomize_cr_path is defined"
+
+- name: Ensure directory exists
+  ansible.builtin.file:
+    path: "{{ cifmw_edpm_kustomize_cr_path | dirname }}"
+    state: directory
+
+- name: Check if there is already a kustomization
+  register: cifmw_edpm_kustomize_kustomize_yaml_stat
+  ansible.builtin.stat:
+    path: "{{ cifmw_edpm_kustomize_cr_path | dirname }}/kustomization.yaml"
+
+- name: Update the already existing kustomization to point to the new CR
+  when: cifmw_edpm_kustomize_kustomize_yaml_stat.stat.exists
+  block:
+    - name: Load the kustomization
+      ansible.builtin.slurp:
+        src: "{{ cifmw_edpm_kustomize_cr_path | dirname }}/kustomization.yaml"
+      register: cifmw_edpm_kustomize_yaml_slurp
+
+    - name: Prepare and load the ci-framework kustomize template file
+      vars:
+        _kustomize_content: "{{ cifmw_edpm_kustomize_yaml_slurp['content'] | b64decode | from_yaml }}"
+        _resource_list: >-
+          {{
+            kustomize_content.resources |
```

Comment:
I was going through the pr. I can see `_kustomize_content` is defined but not `kustomize_content` var. I am not sure where it is coming from, can you point it to the correct line. Rest looks good.

I am asking this because https://logserver.rdoproject.org/25/425/29cc3e7c9562c19fa836b9ac06529b3e48847b81/github-check/cifmw-molecule-edpm_kustomize/416205c/report.html
```
TASK [edpm_kustomize : Prepare and load the ci-framework kustomize template file] ***
TASK [edpm_kustomize : Prepare and load the ci-framework kustomize template file] ***
skipping: [instance]
skipping: [instance]
```

Reply:
dang, perfectly right. And I could improve a bit the whole thing, allowing to remove some code duplication - thanks @raukadah !